src/styles/blocks/explore.scss

```diff
@@ -0,0 +1,73 @@
+.explore {
+
+  &__activities {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    justify-items: center;
+
+    @include onDesktop {
+      grid-template-columns: repeat(auto-fill, minmax(310px, 1fr));
+      gap: $gridGap;
+      align-items: start;
+    }
+  }
+
+  &__activity {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: min-content;
+    margin-bottom: 48px;
+
+    &--link {
+      text-decoration: none;
+    }
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+
+  &__title {
+    margin-bottom: 40px;
+  }
+
+  &__subtitle {
+    margin: 24px 0 4px;
```

Comment:
don't need to hardcode each pixel. Don't forget about the default margins and paddings